templates/image-builder.yml

```diff
@@ -34,6 +34,64 @@ objects:
     selector:
       matchLabels:
         name: image-builder
+    strategy:
+      type: Rolling
+      rollingParams:
+        pre:
+          failurePolicy: Abort
+          execNewPod:
+            containerName: image-builder
+            command: [ "/app/image-builder-migrate-db" ]
+            env:
```

Comment:
I am not sure if execNewPod needs the env specified a second time. This instance of the container definitely needs the additional `MIGRATIONS_DIR` defined, but other than that it shares most variables with the main container.